curwmysqladapter/pymysqlpool.py

```diff
@@ -0,0 +1,61 @@
+import threading
+import pymysql
+
+
+def new_close(conn):
+    if conn.pooling is not None:
+        conn.pooling.put(conn)
+    elif conn.old_close is not None:
+        conn.old_close()
+
+class Pool(object):
```

Comment:
@madumalt did you write this code along ? Seems to be it's awesome. Why aren't you contributing this code to pymysql ?